downloads.markdown

```diff
@@ -159,3 +122,12 @@ $$ git clone https://github.com/haskell/network
 $$ cd network
 $$ cabal install
 ```
+# You need Help?
+
+### Where to get help<a name="help"></a>
+
+*   For help learning Haskell itself, start with the [Documentation](https://www.haskell.org/documentation) page on the [Haskell Wiki](https://wiki.haskell.org/).
+*   If you need help with [GHC](https://www.haskell.org/ghc)---the Haskell compiler---there is a comprehensive [GHC User Manual](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/index.html).
+*   For help using Cabal to download or create additional packages (see [below](#libraries)), there is the [Cabal User Guide](https://www.haskell.org/cabal/users-guide/).
+*   For help using Stack to download or create packages, see the stack documentation [below](#stackhelp).
+*   Finally, you can ask questions of other Haskell users and experts on the [#haskell IRC channel](irc://irc.freenode.net/haskell) on the Freenode IRC network.
```

Comment:
IMO, it would be better to have the IRC link be a link to http://webchat.freenode.net/?randomnick=1&channels=%23haskell&uio=d4